packages/@aws-cdk/aws-codepipeline/lib/cross-region-support-stack.ts

```diff
@@ -2,6 +2,25 @@ import kms = require('@aws-cdk/aws-kms');
 import s3 = require('@aws-cdk/aws-s3');
 import cdk = require('@aws-cdk/core');
 
+export class CrossRegionSupportConstruct extends cdk.Construct {
```

Comment:
Remove the "Construct" posfix: `CrossRegionSupport`

Reply:
That I can't do: that name is already taken (by the customer-facing interface that's returned from the `Pipeline`'s class `crossRegionSupport` method). `CrossRegionSupportConstruct` is private to the `codepipeline` package.